content/docs/studio/user-guide/projects-and-experiments/run-experiments.md

```diff
@@ -16,24 +16,108 @@ your model training using Iterative Studio.
 
 Iterative Studio uses your regular CI/CD setup (e.g. GitHub Actions) to run the
 experiments. This means that to enable experimentation from Iterative Studio,
-you need to integrate your Git repository with a CI/CD setup (e.g. GitHub
-Actions).
+you need to do the following:
 
-Then, on each Git commit, the CI/CD setup will get invoked. If this setup
-includes a model training process, it will be triggered, which means that your
-ML experiment will run automatically.
+1. First, integrate your Git repository with a CI/CD setup that includes model
+   training process. You can do this in one of two ways:
```

Comment:
model training process -> job execution.

Reply:
This has been changed to link to the prepare-repositories page. PTAL.